torch/ao/quantization/experimental/observer.py

```diff
@@ -133,10 +136,16 @@ def forward(self, x_orig):
         self.max_val = max_val
         return x_orig
 
-    def quant_levels_visualization(self, obs_result, filename):
+    r"""Displays visualization of APoT quantization levels
+        Args:
+            alpha: alpha APoT qparam (Tensor)
+            quantization_levels: quantization levels APoT qparam (Tensor)
+            level_indices: level indices APoT qparam (Tensor)
+    """
+    def quant_levels_visualization(self, alpha, quantization_levels, level_indices):
```

Comment:
maybe there is some misunderstanding, what I meant is not to have these args, but get them from calculate_qparams:
```
alpha, gamma, quantization_levels, level_indices = self.calculate_qparams(...)
```

Reply:
sorry i made that change but accidentally didn't include in my last push, should be correct now!